### ISSUE TITLE
RangeError after executing flutter pub run import_sorter:main

### ISSUE THREAD

**Issue by KevinRohn** [fluttercommunity/import_sorter#37]
**Describe the bug**
After executing the `flutter pub run import_sorter:main` command I get following error:

```
username@PC044-MacOS flutter_my_app % flutter pub run import_sorter:main

┏━━🗂  Sorting 94 dart files
┃  ┃
┃  ┣━━ ✅ Sorted 0 out of 7 imports in lib/drawer/drawer_navigation.dart
┃  ┣━━ ✅ Sorted 0 out of 1 imports in lib/drawer/naviagtion_router.dart
Unhandled exception:
RangeError (index): Invalid value: Not in inclusive range 0..1: 2
#0      List.[] (dart:core-patch/growable_array.dart:177:60)
#1      main (file:///Users/username/development/flutter/.pub-cache/hosted/pub.dartlang.org/import_sorter-4.4.0/bin/main.dart:97:149)
#2      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
pub finished with exit code 255
```

**To Reproduce**
I installed the `import_sorter` package and did run `flutter pub run import_sorter:main`

**Expected behavior**
correct sorting, but it stopped after two files.

**Screenshots**
![Bildschirmfoto 2020-11-23 um 17 25 56](https://user-images.githubusercontent.com/25847098/99987784-251c3900-2db1-11eb-8c5c-e3a009a6c941.png)

**Meta Information:**
 - Dart Version: `2.10.4`
 - `import_sorter` Version: `4.4.0`
 - Is it a Flutter project? Yes
 - What version of Flutter are you using (if flutter project): `1.22.4`

**Comment by Matt-Gleich** [comment]
Hello @KevinRohn! Sorry to hear you are having an issue with this. Would you mind sending me the file that you are having an error on so I can easily debug this? Thank you!

**Comment by Matt-Gleich** [comment]
Also, do you get this error if you use an older version of `import_sorter`?

**Comment by aseef17** [comment]
Facing the same error with import_sorter 4.4.0

**Comment by aseef17** [comment]
> Also, do you get this error if you use an older version of `import_sorter`?

Works well with 4.2.2

**Comment by Matt-Gleich** [comment]
@aseef17 could you send me the file you are getting an error for?

**Comment by Matt-Gleich** [comment]
I was just able to reproduce this. Looking into a fix now

**Comment by Matt-Gleich** [comment]
Just fixed. Releasing 4.4.1 now

**Comment by Matt-Gleich** [comment]
This is now fixed in `import_sorter` v4.4.1!

**Comment by KevinRohn** [comment]
Thank you!
